zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/callback/ZkAsyncCallMonitorContext.java

```diff
@@ -62,4 +62,18 @@ void recordAccess(String path) {
       }
     }
   }
+
+  /**
+   * Record the operation failure into the specified ZkClient monitor.
+   * @param path The monitored path
+   */
+  void recordFailure(String path) {
+    if (_monitor != null) {
+      if (_isRead) {
+        _monitor.recordFailure(path, ZkClientMonitor.AccessType.READ);
```

Comment:
We could condense the lines to `_isRead ? ZkClientMonitor.AccessType.READ : ZkClientMonitor.AccessType.WRITE`

Reply:
Thanks, @NealSun96 for your suggestion. The reason I didn't write it that way is, as @zhangmeng916 suggested, the function above this one uses the same pattern so I kept it that way for consistency.